[ AI Deploy beta ] - scaling strategies (#3651)
ovh/docs: pages/platform/ai/deploy_guide_04_scaling_strategies/guide.en-asia.md

---
title: AI Deploy - Scaling strategies
slug: deploy/apps-deployments
excerpt: Understand the scaling strategies (static scaling vs autoscaling) of AI Deploy
section: AI Deploy guides
order: 04
---

**Last updated 3rd November, 2022.**

> [!primary]
>
> AI Deploy is in `beta`. During the beta-testing phase, the infrastructure’s availability and data longevity are not guaranteed. Please do not use this service for applications that are in production, as this phase is not complete.
>
> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
>

## Objective

> This guide covers the use of the different scaling strategies for AI Deploy. The aim is to understand the difference between **static scaling** and **autoscaling** to be able to choose the best solution depending on the use case and type of deployment.

Reviewer: ```suggestion
This guide covers the use of the different scaling strategies for AI Deploy. It aims to understand the difference between **static scaling** and **autoscaling** to be able to choose the best solution depending on the use case and type of deployment.
```
